app/helpers/proposals_helper.rb

```diff
@@ -22,4 +22,12 @@ def status_icon_tag(status, last_approver = false)
       style: ("background-image: url('#{bg_linear_image}');" unless last_approver)
     )
   end
+
+  def get_new_feature_image_tag(new_feature_date)
+    if cookies[new_feature_date]
```

Comment:
I think it'd be very cool to refactor this expression, to help show the intention. E.g.:

```ruby
  if new_feature_is_active
  ...


  def new feature_is_active
    cookies[new_feature_date]
  end
```